### PR TITLE
438: Merge Web-UI Back

### LINKED ISSUE #438: Merge Zally and Zally-Web-UI Repositories
**As** Zally maintainer
**I want** to have the source code of all components in one place
**So that** we can implement richer integration tests
**And** we can provide easier and better getting started guides and scripts

**Acceptance Criteria**
* [Zally Web UI](https://github.com/zalando-incubator/zally-web-ui) source code is moved to this repository.
* [Zally Web UI](https://github.com/zalando-incubator/zally-web-ui) links to the merged repository
* Travis configuration is adjusted and works
* Root README.md describes the project and all its components

### PR DESCRIPTION
**Pull request by netme**
Closes #438 

⚠️ do not merge

This is a first attempt to merge https://github.com/zalando-incubator/zally-web-ui back to this repo keeping the whole history. 

### TODO:
- [x] Move web-ui related files from `src` folder to `web-ui`
- [x] Try to launch project
- [x] Setup CI back

### CODE REVIEW

**Comment by netme** [comment]
@rbarilani @EduardoAC could you please check if the project is still looking good?

**Comment by maxim-tschumak** [comment]
@netme there are still some files in `src/` directory which are not moved to `web-ui`.

**Comment by netme** [comment]
@maxim-tschumak thanks for checking. Could you please give few examples?

**Comment by netme** [comment]
Ah, now I see that :-) Will fix that

**Comment by netme** [comment]
Seems to be everything is back to normal. 
- CI is updated
- Paths are now correct

@maxim-tschumak, @rbarilani could you please check?

File: web-ui/.eslintignore [hunk 1]
```diff
@@ -0,0 +1 @@
+src/client/public
```
Comment (EduardoAC) [inline web-ui/.eslintignore]: Missing new line

**Comment by maxim-tschumak** [comment]
Looks good to me!
:+1:

**Comment by rbarilani** [comment]
👍

**Comment by EduardoAC** [comment]
👍

**Comment by netme** [comment]
👍

**Comment by netme** [comment]
@EduardoAC I will open another PR fixing links, it's becoming tricky to keep this one up to date

**Comment by EduardoAC** [comment]
👍

**Pull request merged**